### ISSUE TITLE
Firefox Developer Edition Support

### ISSUE THREAD

**Issue by HaoZeke** [tridactyl/tridactyl#1099]
*   Brief description of the problem: Cannot get `native` to work.

*   Steps to reproduce:

1. Install Firefox Developer Edition
2.  Install Tridactyl
3. Run `:installnative`
![image](https://user-images.githubusercontent.com/4336207/47295864-59e7c000-d62e-11e8-8743-0c64f4c82c59.png)
4. Run `:native`
5. Be confused
![image](https://user-images.githubusercontent.com/4336207/47295846-476d8680-d62e-11e8-9c40-ef13ec1578c4.png)

*   Tridactyl version (`:version`): 1.14.0-1.14.0

*   Firefox version (Top right menu > Help > About Firefox): 

![image](https://user-images.githubusercontent.com/4336207/47295681-df1ea500-d62d-11e8-97b1-5368ee30b1c5.png)

*   URL of the website the bug happens on: N/A

*   Config (in a new tab, run `:viewconfig`, copy the url and paste it somewhere like gist.github.com):

*   Contents of ~/.tridactylrc or ~/.config/tridactyl/tridactylrc (if they exist):

```
Insert tridactylrc contents between the backticks
```

*   Operating system: Arch Linux
![image](https://user-images.githubusercontent.com/4336207/47295707-f2ca0b80-d62d-11e8-8403-1c3e03bfb2e2.png)

*   Result of running `:! echo $PATH`, or `! echo %PATH%` on Windows, in Tridactyl:
*   Unix-like only: result of running `printf '%c\0\0\0{"cmd": "run", "command": "echo $PATH"}' 39 | ~/.local/share/tridactyl/native_main.py` in a terminal:

```bash
printf '%c\0\0\0{"cmd": "run", "command": "echo $PATH"}' 39 | ~/.local/share/tridactyl/native_main.py
&{"cmd": "run", "content": "/home/haozeke/.zplug/bin:/home/haozeke/Git/Github/Dotfiles/Scripts:/home/haozeke/.node_modules/bin:/home/haozeke/.yarn/bin:/home/haozeke/.local/bin:/home/haozeke/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/opt/cxoffice/bin:/var/lib/flatpak/exports/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/var/lib/snapd/snap/bin:/home/haozeke/.rvm/bin:/home/haozeke/.local/bin/cronDrivers:/home/haozeke/.local/bin/cryptHelpers:/home/haozeke/.local/bin/dockerHelpers:/home/haozeke/.local/bin/droidHelpers:/home/haozeke/.local/bin/fileHelpers:/home/haozeke/.local/bin/gitHelpers:/home/haozeke/.local/bin/installHelpers:/home/haozeke/.local/bin/shellHelpers:/home/haozeke/.local/bin/webHelpers:/home/haozeke/.cargo/bin:/home/haozeke/.go/bin:/home/haozeke/Git/Github/Linux/xmr-stak-cpu/bin://home/haozeke/.gem/ruby/2.5.0/bin:/home/haozeke/LabWork/IITB/Icover:/root/.gem/ruby/2.2.0/bin:/home/haozeke/Matlab/MATLAB_Production_Server/R2015a/bin:/home/haozeke/.gem/ruby/2.2.0/bin:/home/haozeke/.gem/ruby/2.3.0/bin:/opt/schrodinger2016-2:/home/haozeke/Git/Github/mylammps/src:/home/haozeke/Git/Github/espresso/build:/home/haozeke/Git/Github/Scripts/Build:/home/haozeke/Git/Github/Scripts/System:/home/haozeke/.local/g16:/usr/local/MATLAB/R2017a/bin\n", "code": 0}%     
```

Basically I am unable to get `:native` to work with the Developer Edition.

**Comment by glacambre** [comment]
Hi, thank you for opening this issue. Isee flatpak and snaps mentionned in your installed packages, is firefox one of these? Also are you perhaps running Firefox in Firejail?

**Comment by HaoZeke** [comment]
No, my firefox is straight from the standard repositories of arch..
![image](https://user-images.githubusercontent.com/4336207/47338142-6d3d6e80-d6b4-11e8-813f-bfa5986911e7.png)

![image](https://user-images.githubusercontent.com/4336207/47338163-821a0200-d6b4-11e8-9d1f-851da386c719.png)

**Comment by HaoZeke** [comment]
Sure.. Sorry for the delay:

![image](https://user-images.githubusercontent.com/4336207/47527295-dc3ae300-d8bf-11e8-85e7-bceb885c538d.png)

**Comment by glacambre** [comment]
No problem. I think the issue here is that the manifest doesn't point to the right file. replace `"path": ":/usr/share/tridactyl/native_main.py"` with `"path": "/home/haozeke/.local/share/tridactyl/native_main.py"` and things should start working.

There must be a bug in our install script, I doubt it has been tested with non-standard XDG_DATA_HOMEs

**Comment by HaoZeke** [comment]
Yup that worked!! Thanks a ton @glacambre

**Comment by glacambre** [comment]
You're welcome, thanks for reporting back :)

I'm reopening this issue until we fix our native installer :)
